src/modules/UI/scenes/Otp/OtpSettingsSceneComponent.js

```diff
@@ -54,7 +56,8 @@ export default class OtpSettingsScene extends Component<OtpSettingsSceneProps, S
     }
     this.setState({
       showMessageModal: true,
-      messageModalMessage: s.strings.otp_enabled_modal_part_one + '' + s.strings.otp_enabled_modal_part_two
+      messageModalMessage: s.strings.otp_enabled_modal_part_one + '' + s.strings.otp_enabled_modal_part_two,
```

Comment:
The `messageModalMessage` state seems unnecessary here as only the `messageModalComponent` would get rendered. If so, set to NULL